### DISCUSSION TITLE
Permissions

### DISCUSSION THREAD

**Discussion by ocularvr** [mozilla/hubs#2831] [General Questions & Discussion] [answered]
Hi, I would like my users to access the hubs room we have created but without the possibility of sharing anything besides the video/audio feed. How can I start to do this? I changed all the parameters in the back end but no success. I cant find any documentation on how to control what a user can and cant do.

**Comment by brianpeiris** [comment, marked as answer]
Hi @ocularvr. You can control the permissions in a hubs room from the "Room Settings" dialog, which you can access from the room menu in the top left corner. You can use these settings to control what particular abilities your room's visitors will have:
![image](https://user-images.githubusercontent.com/79419/89825482-81f92980-db09-11ea-8cca-b07841dfd7b6.png)

**Reply by ocularvr** [reply]
Thanks Brian! I know this, what I cant get is the ability to share the video feed but without being able to place 3d objects or draw. It seems that with camera controls comes the other stuff. I want to allow the user to share audio and video but not share 3d objects or draw.

**Reply by brianpeiris** [reply]
Ah, I see. Unfortunately there isn't an easy way for us to distinguish 3d object media from video media at the moment, because of the way things work internally. Feel free to create a github issue to request that ability. I don't believe we have an issue for that particular scenario yet. Definitely appreciate the feedback.
